dbms/src/DataStreams/IProfilingBlockInputStream.cpp

```diff
@@ -120,8 +131,7 @@ void IProfilingBlockInputStream::readPrefix()
     auto start_time = info.total_stopwatch.elapsed();
     readPrefixImpl();
 
-    forEachChild([&] (IBlockInputStream & child)
-    {
+    forEachChild([&](IBlockInputStream & child) {
```

Comment:
should we add lock for `info.updateExecutionTime(info.total_stopwatch.elapsed() - start_time);`?